src/Backend/Core/Layout/Sass/components/_tabs.scss

```diff
@@ -77,3 +77,11 @@
     background-color: $white;
   }
 }
+
+.tab-pane-collection {
+  margin: -31px -26px -56px;
```

Comment:
Those are some very specific numbers

Reply:
they are needed to cancel out the padding and margin of the tab container